src/Rule/ContainsUppercaseCharacters.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Morebec\Validator\Rule;
+
+
+use Morebec\Validator\ValidationRuleInterface;
+
+class ContainsUppercaseCharacters implements ValidationRuleInterface
+{
+    /**
+     * @var int
+     */
+    private $numberCharacters;
+
+    /**
+     * @var bool
+     */
+    private $strict;
+    /**
+     * @var string|null
+     */
+    private $message;
+
+    /**
+     * ContainsUppercaseCharacters constructor.
+     * @param int|null $numberCharacters
+     * @param bool|null $strict
+     * @param string|null $message
+     */
+    public function __construct(
+        ?int $numberCharacters = 1,
```

Comment:
Can you validate that `numberCharacters` is a positive integer or zero?
Also, why did you make it accept null?